src/compressed_tensors/offload/cache/disk.py

```
@@ -169,11 +153,25 @@ def create_checkpoint_symlink(
+        finalize(offloaded, cls._disk_finalizer, id(offloaded))
+
+    @classmethod
+    def _disk_finalizer(cls, tensor_id: int):
+        """
+        Finalizer attached to tensors when they are assigned in `DiskCache.index`.
+        Deletes tensor from `DiskCache.index` and deletes associated safetensors file.
+
+        :param tensor_id: id of offloaded meta tensor
+        """
+        file_path = cls.index[tensor_id]["safetensors_file"]
+        assert _is_ct_file_path(file_path)
```

Ruff (0.15.15)

[error] Undefined name `_is_ct_file_path`

(F821)